tests/test_one_client_read_loop_server.py

```diff
@@ -0,0 +1,84 @@
+# This file is part of ts_tcpip.
+#
+# Developed for the Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asyncio
+import logging
+import unittest
+
+from lsst.ts import tcpip
+
+# Standard timeout in seconds.
+TIMEOUT = 60
+
+
+class OneClientReadLoopServerTestCase(unittest.IsolatedAsyncioTestCase):
+    async def asyncSetUp(self) -> None:
+        self.srv = tcpip.TestOneClientReadLoopServer()
+        self.log = logging.getLogger(type(self).__name__)
+
+        await self.srv.start_task
+        assert self.srv.server.is_serving()
+        self.reader, self.writer = await asyncio.open_connection(
+            host=self.srv.host, port=self.srv.port
+        )
```

Comment:
Please consider using the Client class, as it offers a useful example and allows use of matching methods.

I suspect you'll want an async context manager to create this stuff -- asyncSetUp is unfortunately susceptible to hanging.

Reply:
The change to an async context manager for the client works wonderfully well. Thanks for the tip!

Reply:
I also expanded the test to send 5 messages, instead of 1, to ensure that the read loop keeps on running after the first message.